src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -0,0 +1,221 @@
+			}
+
+			cl, err := adminapi.NewClient(fs, p)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			response, err = cl.Reconfigurations(cmd.Context())
+			out.MaybeDie(err, "unable to list partition movements: %v\n", err)
+
+			if len(response) == 0 {
+				out.Exit("Currently there's no ongoing partition movements.")
+			}
+
+			if len(args) > 0 && len(partitions) > 0 { // yes topic && yes partition
+				j := 0
+				for _, t := range args {
+					if strings.Count(t, "/") == 0 {
+						for _, p := range partitions {
+							for _, res := range response {
+								if res.Ns == "kafka" && res.Topic == t && strconv.Itoa(res.PartitionID) == p {
+									response[j] = res
+									j++
+								}
+							}
+						}
+					} else if strings.Count(t, "/") == 1 {
+						nt := strings.Split(t, "/")
+						for _, p := range partitions {
+							for _, res := range response {
+								if res.Ns == nt[0] && res.Topic == nt[1] && strconv.Itoa(res.PartitionID) == p {
+									response[j] = res
+									j++
+								}
+							}
+						}
+					} else {
+						fmt.Printf("invalid format for topic %s, skipping.\n", t)
+					}
+				}
+				response = response[:j]
+			} else if len(args) > 0 && len(partitions) <= 0 { // yes topic && no partition
+				j := 0
+				for _, t := range args {
+					if strings.Count(t, "/") == 0 {
+						for _, res := range response {
+							if res.Ns == "kafka" && res.Topic == t {
+								response[j] = res
+								j++
+							}
+						}
+					} else if strings.Count(t, "/") == 1 {
+						nt := strings.Split(t, "/")
+						for _, res := range response {
+							if res.Ns == nt[0] && res.Topic == nt[1] {
+								response[j] = res
+								j++
+							}
+						}
+					} else {
+						fmt.Printf("invalid format for topic %s, skipping.\n", t)
+					}
+				}
+				response = response[:j]
+			}
+
+			sizeFn := func(size int) string {
+				if human {
+					return units.HumanSize(float64(size))
+				}
+				return strconv.Itoa(size)
+			}
+
+			f := func(rr *adminapi.ReconfigurationsResponse) interface{} {
+				var (
+					newReplica []int
+					oldReplica []int
+				)
+				nt := rr.Ns + "/" + rr.Topic
+				if rr.PartitionSize > 0 {
+					completion = rr.BytesMoved * 100 / rr.PartitionSize
+				}
+				for _, r := range rr.NewReplicas {
+					newReplica = append(newReplica, r.NodeID)
+				}
+				for _, r := range rr.PreviousReplicas {
+					oldReplica = append(oldReplica, r.NodeID)
+				}
+				return struct {
+					NT             string
+					PartitionID    int
+					MovingFrom     []int
+					MovingTo       []int
+					Completion     int
+					PartitionSize  string
+					BytesMoved     string
+					BytesRemaining string
+				}{
+					nt,
+					rr.PartitionID,
+					oldReplica,
+					newReplica,
+					completion,
+					sizeFn(rr.PartitionSize),
+					sizeFn(rr.BytesMoved),
+					sizeFn(rr.BytesLeft),
+				}
+			}
+
+			types.Sort(response)
+
+			out.Header("Ongoing partition movements", true, true, func() {
```

Comment:
[question] Will we show any **not** ongoing partition movements?

If no, I think we can change `Ongoing partition movements` -> `partition movements`